joinservice/cmd/main.go

```diff
@@ -49,35 +51,39 @@ func main() {
 	verbosity := flag.Int("v", 0, logger.CmdLineVerbosityDescription)
 	flag.Parse()
 
-	log := logger.New(logger.JSONLog, logger.VerbosityFromInt(*verbosity))
+	log := slog.New(slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{Level: logger.VerbosityFromInt(*verbosity)}))
```

Comment:
While choosing the right log output stream is a [quite controversal topic](https://github.com/ipfs/go-log/issues/60#issuecomment-484140608), I'd personally advocate for writing to stderr. Would like to hear more opinions on this though.

Reply:
I'm also in favor of stderr. What did the old implementation write to?